### PR TITLE
Update how-to-style-a-row-in-a-listview-that-implements-a-gridview.md

### PR DESCRIPTION
**Pull request by wenhx**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/framework/wpf/controls/how-to-style-a-row-in-a-listview-that-implements-a-gridview.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ helpviewer_keywords:
 ms.assetid: 2e406ba2-70a0-4e62-841f-0934859de76e
 ---
 # How to: Style a Row in a ListView That Implements a GridView
-This example shows how to style a row in a <xref:System.Windows.Controls.ListView> control that implements a <xref:System.Windows.Controls.GridView><xref:System.Windows.Controls.ListView.View%2A> mode.  
+This example shows how to style a row in a <xref:System.Windows.Controls.ListView> control that implements a <xref:System.Windows.Controls.GridView> <xref:System.Windows.Controls.ListView.View%2A> mode.
```
Comment (Youssef1313) [inline docs/framework/wpf/controls/how-to-style-a-row-in-a-listview-that-implements-a-gridview.md:11]: I think it's usually said that "an interface is implemented" but "a class is inherited".

The above sentence uses implements, but GridView is a class.

I also feel the sentence is missing something between `GridView` and `View`?

Comment (adegeo) [reply docs/framework/wpf/controls/how-to-style-a-row-in-a-listview-that-implements-a-gridview.md:11]: Yeah it's the wrong word. Really it's just "use a gridview as the view mode" This PR though is really correcting the rendering problem this page has.

It's displaying `... implements a GridViewView` and this fixes it so it at least says `... implements a GridView View`.

The article itself is really about styling a row in a listview that is using a gridview view.

I'll edit the PR to adjust that and approve.

Thanks @wenhx

**Review by adegeo — APPROVED** [review]

**Comment by adegeo** [comment]
Thanks @wenhx

**Comment by wenhx** [comment]
My pleasure. @adegeo

**Pull request merged**